src/pages/organize/[orgId]/campaigns/[campId]/surveys/[surveyId]/submissions.tsx

```diff
@@ -46,16 +53,36 @@ const SubmissionsPage: PageWithLayout<SubmissionsPageProps> = ({
       new SurveySubmissionsModel(env, parseInt(orgId), parseInt(surveyId))
   );
 
+  const campaignId = isNaN(parseInt(campId)) ? 'standalone' : parseInt(campId);
```

Comment:
`NaN` is falsy, so this could also be written `const campaignId = parseInt(campId) || 'standalone'`.